Imabulary/Imabulary
# [Feature] [FE] Payments processing foundation (IMB-58)

## Overview
This PR will add a wallet screen and ability to collect daily award from home screen

## Test cases

1. Go to Profile screen and pay attention to left upper corner of the screen, there you must see wallet's balance and a "More Coins" button
2. Click on the "More Coins" button and you must be redirected to the Wallet screen

## Issue

closes [58](https://www.notion.so/Implement-the-FE-part-of-all-features-40918fdcefaa40e7a48995097d90658b?pvs=4)
